### ISSUE TITLE
Constantly blinking irplus remote instead of clicking each button. like on the concert. all different colors

### ISSUE THREAD

**Issue by theimpossibleheir** [danielweidman/pixmob-ir-reverse-engineering#50]
I'm not familiar with this but i downloaded the google drive, and the remote works but only to click the colors separately. would you be able to create a one button that will synchronize all colors like changing it like in the concert?

**Comment by sean1983** [comment]
Not really!   You can create Macros inside of IR Plus so maybe do something using them
